### PR TITLE
Adds commit sha to lua display

### PR DESCRIPTION
**Pull request by JyeSmith**
Adds a 6 figure short sha to the lua display.  This can be used to determine the branch being flown by using a link like below.

`
https://github.com/AlessandroAU/ExpressLRS/commit/a4269a`

### CODE REVIEW

File: src/lua/ExLRS.lua [hunk 1]
```diff
@@ -167,7 +174,8 @@ end
 
 local function refreshHorus()
     lcd.clear()
-    lcd.drawText(1, 1, 'ExpressLRS CFG v.01', INVERS)
+    -- lcd.drawText(1, 1, 'ExpressLRS CFG v.01', INVERS)
```
Comment (wvarty) [inline src/lua/ExLRS.lua:177]: Still need this commented out line?

File: src/lua/ExLRS.lua [hunk 2]
```diff
@@ -196,7 +204,8 @@ end
 
 local function refreshTaranis()
     lcd.clear()
-    lcd.drawScreenTitle('ExpressLRS CFG ' .. version, 1, 1)
+    -- lcd.drawScreenTitle('ExpressLRS CFG ' .. version, 1, 1)
```
Comment (wvarty) [inline src/lua/ExLRS.lua:207]: Same as above

File: src/lib/CRSF/CRSF.cpp [hunk 3]
```diff
@@ -192,10 +193,10 @@ void ICACHE_RAM_ATTR CRSF::sendLUAresponse(uint8_t val1, uint8_t val2, uint8_t v
     outBuffer[3] = CRSF_ADDRESS_RADIO_TRANSMITTER;
     outBuffer[4] = CRSF_ADDRESS_CRSF_TRANSMITTER;
 
-    outBuffer[5] = val1;
-    outBuffer[6] = val2;
-    outBuffer[7] = val3;
-    outBuffer[8] = val4;
+    for (uint8_t i = 0; i < dataLength; ++i)
+    {
+        outBuffer[5+i] = val[i];
```
Comment (wvarty) [inline src/lib/CRSF/CRSF.cpp:198]: Be nice to have some spaces between arithmetic, but that just me :)

**Comment by JyeSmith** [comment]
Thanks @wvarty 👍

**Pull request merged**